python/plugins/processing/ui/DlgScriptEditor.ui

```diff
@@ -155,6 +156,14 @@
     <string>Ctrl+Shift+S</string>
    </property>
   </action>
+  <action name="actionLoadFromTemplate">
+   <property name="text">
+    <string>Load script from template...</string>
```

Comment:
"Load Script from Template…" (note ellipsis character on the end, not three dots)